webapp/src/utils/validators.ts

```diff
@@ -42,6 +42,13 @@ const requiredStringListValidator = stringListValidator(true)
 const optionalStringListValidator = stringListValidator(false)
 
 const numericStringValidator = (value: string): BasicValidation => {
+    //unluckily 0 is null for js/type script
+    //unluckily, 0 is still number
+    //this validator do not eat null as 0
+    if (value == null) {
```

Comment:
based on parameter type, value cannot be null, only string...so you need to check if value is empy string

Reply:
Which is already done later.
I know I could had skipped the check, and directly call the value.toString(), but I somhow did not trusted that.
Do you really wish to remove the null chek, and proceed directly with value.toString()?

Reply:
if you think null check is necessary, then value should by of type string | null